RockLib.Messaging.CloudEvents/CloudEvent.cs

```diff
@@ -628,10 +746,14 @@ public virtual void Validate()
             _ = Id;
 
             if (Source is null)
+            {
                 throw new CloudEventValidationException("Source cannot be null.");
+            }
 
             if (string.IsNullOrEmpty(Type))
```

Comment:
Just a FYI, I think we should always use `IsNullOrWhitespace()` instead. I realize you didn't make this, just a thing we should consider.

Reply:
Updated.